README.md

```diff
@@ -31,6 +31,11 @@ To build RustPython locally, first, clone the source code:
 git clone https://github.com/RustPython/RustPython
 ```
 
+If on windows, run the following to fix the `Lib/` directory symlink:
```

Comment:
```suggestion
RustPython uses symlinks to manage python libraries in `Lib/`. If on windows, run the following helps:
```

Does this make sense?

Also, as I know windows developer mode allows user to create symbolic links. Do you know if it also can be an alternative option of this?

Reply:
I'm not sure, this solution worked very well for me though.

Reply:
@coolreader18 how about you? Is mentioning developer mode symbolic links support a correct description?

Reply:
I think this makes sense, I'm pretty sure setting `core.symlinks` makes git make real symlinks using developer mode.